piyushgarg-dev/piyushgargdev-nextjs
# fix: changed the color and logo of socials

## What does this PR do?

This PR updates the colors and logo for the project as part of fixing issue #1697. It also includes adjustments to ensure the new design is cohesive with the existing UI.

Fixes #1697

https://github.com/user-attachments/assets/1daf4e2a-f22e-4772-8efd-5074cfb8c0cc



https://github.com/user-attachments/assets/105e6f5a-371b-4652-8ee1-73a03f4d0812


NOTE : The after changes are made on the same website but the owner has changed the position of the socials .

## Type of change

Bug fix (non-breaking change which fixes an issue)


## How should this be tested?

 Test A: Verify that the new logo appears correctly across all screens.
 Test B: Check if the updated colors apply properly and consistently in the header and footer.
